bin/ck8s

```diff
@@ -43,13 +43,15 @@ SYNC=""
 KUBECTL=""
 GEN_NEW_SECRETS=""
 CONCURRENCY="--concurrency=8"
+CK8S_IPV6_ENABLED=""
 
 for arg in "${@}"; do
   case "${arg}" in
   "--sync") SYNC="sync" ;;
   "--kubectl") KUBECTL="kubectl" ;;
   "--generate-new-secrets") GEN_NEW_SECRETS="--generate-new-secrets" ;;
   "--concurrency="*) CONCURRENCY="${arg}" ;;
+  "--ipv6") export CK8S_IPV6_ENABLED="true" ;;
```

Comment:
Instead of exporting it here please pass it as an argument to `update-ips.bash` for now to align with the other flags.